packages/ontario-frontend-cli/commands/ontario-create-app/handleCreateApp.js

```diff
@@ -162,6 +166,8 @@ async function handleCreateAppCommand(cmd = {}) {
       askProjectName: !options.projectName,
       askEnPage: !options.enPage,
       askFrPage: !options.frPage,
+      askESLint: options.addESLint === undefined,
+      askPrettier: options.addPrettier === undefined,
```

Comment:
is undefined the only value that could happen here (no `null`)?

Maybe just a quick comment about how we expect `false` as a value so we can't check for a falsey value like we do with the fields above.

Reply:
Thanks. That is a good catch, I updated the comments to hopefuly make it clearer. 

For addESLint and addPrettier, we need to know if the user explicitly provided true or false. If they did, we use that value. If not, we set it to undefined to indicate that the question should be asked. Using undefined is intentional here because it clearly indicates that the value has not been set, which is different from null (which might imply an intentional absence of value). These values are set a few lines above, so they cannot be null. They can only be true, false, or undefined.